client/js/components/supporter-address-form.es6

```diff
@@ -81,7 +81,10 @@ function view(state) {
       ])
     ])
   , h('input', {props: {type: 'hidden', name: 'id', value: supporter.id}})
-  , button(R.pick(['loading$', 'error$'], state))
+  , button({
+    ...(state.loading$ && { loading$: state.loading$ }),
+    ...(state.error$ && { error$: state.error$ }),
```

Comment:
Does this work? I can't get it to work in the console.

Alternatively, feel free to use a function from lodash if it's easier.

Reply:
it should work if I did it right, it's a pattern I've used elsewhere for inline optional object properties, but I can look at lodash

Reply:
Huh, this may have been where I found it first, not certain though:
https://medium.com/@slamflipstrom/conditional-object-properties-using-spread-in-javascript-714e0a12f496

I do see some people saying it's safer to use a ternary with null than && though

Reply:
It's working for me in the console

![Screenshot 2024-12-13 at 5 17 28 PM](https://github.com/user-attachments/assets/0f2d111a-d04e-4b97-b360-940c93c39567)

Reply:
lodash has `_.pick` though, so I can just use that, it's definitely more concise for these situations